api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1207 @@
+
+    In the high level **Managed API** the consumption of Events proceeds via the establishment of a named
+    **Subscription** to an EventType. Subscriptions are persistent relationships from an Application (which
+    might have several instances) and the stream of an EventType, whose consumption tracking is managed by
+    Nakadi, freeing Consumers from any responsibility in tracking the current position on a Stream. **Note**
+    *Currently the high level API is out of scope in this specification. It is in the short term plan to be
+    included.*
+
+
+    Scope and status of this document
+    ---------------------------------
+
+    The present API specification is in **draft** state and is subject to change.
+
+    In this document, ready for review are included:
+    * The Schema Registry API, including configuration possibilities for the Schema, Validation, Enrichment
+    and Partitioning of Events, and their effects on reception of Events.
+
+    *  The stantardised event format (see definition of Event, BusinessEvent and DataChangeEvent) (Note: at a
+    later moment this will be configurable and not be inherent part of this API).
+
+    * Topic API (low level).
+
+    Other aspects of the Event Bus are at this moment to be defined and otherwise specificied, not included
+    in this version of this specification.
+
+    Notable omissions here are:
+    * The contract between Nakadi and clients in the context of the high-level API (i.e. the process for
+    clients to establish subscriptions).
+
+    * Submission of events to topics in batched manner.
+
+    * Validation and enrichment procedures.
+
+    * Security scopes (OAuth) for the different operations
+
+    * Validation and Enrichment Strategies available in Nakadi are out of this document an can be consulted
+    in an external document (TBD link to document).
+
+
+  version: '0.5'
+  contact:
+    name: Team Aruha @ Zalando
+    email: team-aruha+nakadi-maintainers@zalando.de
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  oauth2:
+    type: oauth2
+    flow: implicit
+    authorizationUrl: 'https://auth.example.com/oauth2/tokeninfo'
+
+paths:
+  /metrics:
+    get:
+      tags:
+        - monitoring
+      summary: Get monitoring metrics
+      responses:
+        '200':
+          description: Ok
+          schema:
+            $ref: '#/definitions/Metrics'
+        '401':
+          description: Client is not authenticated
+          schema:
+            $ref: '#/definitions/Problem'
+        '500':
+          description: Server error
+          schema:
+            $ref: '#/definitions/Problem'
+        '503':
+          description: Not available
+          schema:
+            $ref: '#/definitions/Problem'
+  /event-types:
+    get:
+      tags:
+        - schema-registry-api
+      description: Returns a list of all registered `EventType`s
+      parameters:
+        - name: X-Flow-Id
```

Comment:
In new shop it's already more than optional header for troubleshooting. It's only way to bind requests to different fragments related to to one page. We use it widely in shop events payload.